gradle-plugin/it/src/it/extension-configuration/build.gradle

```diff
@@ -41,6 +41,8 @@ kubernetes {
     serviceUrlWaitTimeSeconds = 10
     skipPush = true
     skipTag = true
+    skipBuild = false
+    skipResource = false
```

Comment:
Can you toggle to true here, false is the default value, so having this options as true is more explicit.